src/components/vuestic-components/va-option-list/VaOptionList.demo.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <VbDemo>
+    <VbCard
+      title="Options as array of string"
+    >
+      <va-option-list
```

Comment:
I suggest you provide a minimal demo.
In our case it would be v-model and options I believe.

Reply:
I guess I have done exactly what you are saying.
We have the array of options and v-model changing upon selection. Or am I missing something?